### PR TITLE
Updated DTF description here-and-there

### PR DESCRIPTION
**Pull request by laurentdufour**
Shouldn't there actually be a word or two about the convergence of the DTF?

### CODE REVIEW

File: first-analysis-steps/decay-tree-fitter.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ For example, for the decay
 ```python
 '[D*(2010)+ -> (D0 -> K- K+) pi+]CC'
 ```
-you can make the assumption that the (K- K+) combine to form a D0 with a specific invariant mass. This results in a so called *mass-constraint*. In addition the two kaons should originate from exactly the same point in space. If you know that your data only contains prompt D* candidates, you can constrain them to do come from the primary vertex. Boundary conditions like those are called *vertex-constraints*.
+you can make the assumption that the (K- K+) combine to form a D0 with a specific invariant mass. This results in a so called *mass constraint*. In addition the two kaons should originate from exactly the same point in space. If you know that your data only contains prompt D\* candidates, you can constrain them to do come from the primary vertex. Boundary conditions like those are called *vertex constraints* (the last of which is known as a *primary-vertex constraint*.
```
Comment (vlisovsk) [inline first-analysis-steps/decay-tree-fitter.md]: Don't you miss a closing bracket in the last sentence?

File: first-analysis-steps/decay-tree-fitter.md [hunk 2]
```diff
@@ -12,9 +12,9 @@ For example, for the decay
...
+you can make the assumption that the (K- K+) combine to form a D0 with a specific invariant mass. This results in a so called *mass constraint*. In addition the two kaons should originate from exactly the same point in space. If you know that your data only contains prompt D\* candidates, you can constrain them to do come from the primary vertex. Boundary conditions like those are called *vertex constraints* (the last of which is known as a *primary-vertex constraint*).
 
-Applying such kinematic constraints leads to new best estimates for the track parameters of the final state particles. The process of calculating those is called a *kinematic refit* and the `TupleToolDecayTreeFitter` is the algorithm that performs this task for us.
+Applying kinematic constraints leads to new best estimates for the track parameters of the final-state particles. The process of calculating those is called a *kinematic (re)fit* and the `DecayTreeFitter` is the algorithm that performs this task for us. Access to this tool is provided by the TupleTool with the name `TupleToolDecayTreeFitter`.
```
Comment (alexpearce) [inline first-analysis-steps/decay-tree-fitter.md]: It's also provided by the `DTF_FUN` functor, which I prefer. Do we mention this anywhere?

**Comment by alexpearce** [comment]
> Shouldn't there actually be a word or two about the convergence of the DTF?

Yeah, that's a good idea.

File: first-analysis-steps/decay-tree-fitter.md (hunk 2 above)
Comment (laurentdufour) [reply first-analysis-steps/decay-tree-fitter.md]: No. Go ahead and add it maybe as a note. It attaches nicely to the lesson on LoKi functors.

Comment (alexpearce) [reply first-analysis-steps/decay-tree-fitter.md]: Opend #148.

**Comment by vlisovsk** [comment]
Thanks Laurent!
I think we can merge this, and proceed with further modifications later on.

**Pull request merged**